src/payloads/Attendance.ts

```diff
@@ -1,22 +1,62 @@
-import { IsOptional, IsBoolean, IsNumber, IsInstance } from 'class-validator';
+import {
+  IsOptional,
+  IsBoolean,
+  IsNumber,
+  IsInstance,
+  ValidateNested,
+  IsInt,
+} from 'class-validator';
+import { Type } from 'class-transformer';
+
 import { AppUserEventResponse } from './AppUser';
 import { EventAttendanceResponse } from './Event';
 
+export class AttendanceCheckOffRequest {
+  @IsInt()
+  readonly attendeeId: number;
+
+  @IsInt()
+  readonly officerId: number;
+
+  @IsBoolean()
+  readonly isInductee: boolean;
+
+  @IsNumber()
+  @IsOptional()
+  readonly duration: number;
+}
+
 export class AttendanceResponse {
   @IsInstance(AppUserEventResponse)
-  readonly attendee: AppUserEventResponse;
+  attendee: AppUserEventResponse;
 
   @IsInstance(EventAttendanceResponse)
-  readonly event: EventAttendanceResponse;
+  event: EventAttendanceResponse;
 
   @IsBoolean()
-  readonly isInductee: Boolean;
+  isInductee: Boolean;
 
   @IsInstance(AppUserEventResponse)
   @IsOptional()
-  readonly officer: AppUserEventResponse;
+  officer: AppUserEventResponse;
 
   @IsNumber()
   @IsOptional()
-  readonly duration: number;
+  duration: number;
+}
+
+export class MultipleAttendanceResponse {
+  @ValidateNested({ each: true })
+  @Type(() => AttendanceResponse)
+  attendances: AttendanceResponse[];
+}
+
+export class MultipleAttendanceQuery {
+  @IsBoolean()
+  @IsOptional()
+  unchecked: boolean;
```

Comment:
what's unchecked?

Reply:
Whether an Attendance is checked off or not

Reply:
So if unchecked is true, then we query only the Attendances that are not checked off